Looport/Millennium-Falcon
# Configure NATS

# Description


This PR includes NATS configuration and deployment to k8s cluster.
As part of this PR, I also implemented sending user registered message to NATS from the passport service.

## What Changed

— Added NATS connector to passport nestjs
— Added nestjs NATS local packages
— Added basic page for room in web
— Changed passport docker configuration to use workspace packages

## How To Test


— Run application
— Register user and see in NATS monitor system that `passport.user.registered` event emitted with user data

# Checklist
- [x] I followed all possible cases of the implemented _user flow_
